TypeCobol/Compiler/CodeElements/Expressions/Expression.cs

```diff
@@ -201,20 +257,90 @@ public override bool AcceptASTVisitor(IASTVisitor astVisitor)
 		}
 	}
 
-	/// <summary>
-	/// Relational operators and their meanings
-	/// </summary>
-	public enum RelationalOperator
-	{
-		EqualTo,
-		GreaterThan,
-		GreaterThanOrEqualTo,
-		LessThan,
-		LessThanOrEqualTo,
-		NotEqualTo
-	}
+    public class RelationalOperator : IVisitable
+    {
+        public static RelationalOperator EqualTo = new RelationalOperator(new SyntaxProperty<RelationalOperatorSymbol>(RelationalOperatorSymbol.EqualTo, null));
+        public static RelationalOperator GreaterThan = new RelationalOperator(new SyntaxProperty<RelationalOperatorSymbol>(RelationalOperatorSymbol.GreaterThan, null));
+        public static RelationalOperator GreaterThanOrEqualTo = new RelationalOperator(new SyntaxProperty<RelationalOperatorSymbol>(RelationalOperatorSymbol.GreaterThanOrEqualTo, null));
+        public static RelationalOperator LessThan = new RelationalOperator(new SyntaxProperty<RelationalOperatorSymbol>(RelationalOperatorSymbol.LessThan, null));
+        public static RelationalOperator LessThanOrEqualTo = new RelationalOperator(new SyntaxProperty<RelationalOperatorSymbol>(RelationalOperatorSymbol.LessThanOrEqualTo, null));
+        public static RelationalOperator NotEqualTo = new RelationalOperator(new SyntaxProperty<RelationalOperatorSymbol>(RelationalOperatorSymbol.NotEqualTo, null));
 
-	/// <summary>
+        public Token NotToken { get; private set; }
+        public SyntaxProperty<RelationalOperatorSymbol> RelationalOperatorSyntaxProperty { get; private set; }
```

Comment:
naming: "Operator" should be enough no ?

Reply:
Indeed, it should be named Operator here.
The problem was for the parameter in constructor, as `operator` is a C# keyword.